src/mSupplyMobileApp.js

```diff
@@ -216,8 +245,12 @@ class MSupplyMobileAppContainer extends React.Component {
       isInitialised,
       isLoading,
       syncModalIsOpen,
+      temperatureSyncState,
     } = this.state;
-
+    const sensorSynchroniser = synchroniseSensors.bind(this, {
```

Comment:
this should be bound in constructor pls.

Can also make a const method in mSupplyMobileApp class:

```
const sensorSynchorniser = () = > synchroniseSensor({database: this.databse ....
```

Reply:
f8fbb2bfe3b38313c85b6bdba8af111c9c196357